spec/feature/creating_peep_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'pg'
+
+feature 'Adding a new peep' do
+    scenario 'A user can add a peep' do
+      visit('/peeps')
```

Comment:
This test will pass if it is visit('/new'). As that is the route in app.rb for adding a new peep :).

Good test to check that functionality of the form is working!